### PR TITLE
release_tool: Remove --verify-integration-references option

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
See https://github.com/mendersoftware/mender-qa/pull/527/commits/3dc0af52afdc0c66682fec6993bf2fd9d02b3d9b

File: extra/release_tool.py [hunk 1]
```diff
@@ -3085,72 +3085,6 @@ def do_hosted_release(version=None):
...
-
-        # Try some common locations.
-        paths = ["..", "../go/src/github.com/mendersoftware"]
-        path = find_repo_path(repo.git(), paths)
-
-        if path is None:
-            print("%s not found. Tried: %s" % (repo.git(), ", ".join(paths)))
-            sys.exit(2)
-
-        revs = figure_out_checked_out_revision(None, path)
```
Comment (kacf) [inline extra/release_tool.py:3108]: You can also remove this function. Not used anywhere else, AFAICS.

**Review by kacf — APPROVED** [review]
Pre-approving, but take a look below.

File: extra/release_tool.py (hunk 1 above)
Comment (lluiscampos) [reply extra/release_tool.py:3108]: Good catch, thank you.

**Pull request merged**